src/components/forms/fragments/PostalDetails.tsx

```diff
@@ -1,34 +1,51 @@
 import React from 'react';
 import { Col, Row } from 'reactstrap';
 
-import { Text } from '../Fields';
+import { Select, Text } from '../Fields';
+import { MUNICIPALITIES } from '../../../constants/Municipalities';
+import { useTranslation } from 'react-i18next';
 
-const PostalDetailsFragment = () => (
-  <Row>
-    <Col sm={4}>
-      <Text
-        name={`address`}
-        label={`form.postal_details.field.street_address.label`}
-        placeholder={`form.postal_details.field.street_address.placeholder`}
-        required
-      />
-    </Col>
-    <Col sm={4}>
-      <Text
-        name={`zipCode`}
-        label={`form.postal_details.field.postal_code.label`}
-        placeholder={`form.postal_details.field.postal_code.placeholder`}
-        required
-      />
-    </Col>
-    <Col sm={4}>
-      <Text
-        name={`municipality`}
-        label={`form.postal_details.field.munacipality.label`}
-        placeholder={`form.postal_details.field.munacipality.placeholder`}
-        required
-      />
-    </Col>
-  </Row>
-);
+const PostalDetailsFragment = () => {
+  const { i18n } = useTranslation();
+  return (
+    <Row>
+      <Col sm={4}>
+        <Text
+          name={`address`}
+          label={`form.postal_details.field.street_address.label`}
+          placeholder={`form.postal_details.field.street_address.placeholder`}
+          required
+        />
+      </Col>
+      <Col sm={4}>
+        <Text
+          name={`zipCode`}
+          label={`form.postal_details.field.postal_code.label`}
+          placeholder={`form.postal_details.field.postal_code.placeholder`}
+          required
+        />
+      </Col>
+      <Col sm={4}>
+        <Select
+          name={`municipality`}
+          label={`form.postal_details.field.munacipality.label`}
```

Comment:
Not your fault, but want to quickly rename this `munacipality` key to `municipality`?

Reply:
Done. Also added a placeholder to the select.